fusor-ember-cli/app/templates/review/installation.hbs

```diff
@@ -141,7 +141,7 @@
     {{/if}}
 
     {{#if isOpenStack}}
-      {{#accordion-item name=deploymentController.nameOpenStack isOpen=isOpenStackOpen}}
+      {{#accordion-item name=deploymentController.fullnameOpenStack isOpen=isOpenStackOpen}}
```

Comment:
since you added the alias to review/installation.js, you should be able to delete "deploymentController.".  I'm guessing it was never needed.